keras/backend/torch/optimizers/torch_optimizer.py

```diff
@@ -33,7 +34,11 @@ def __new__(cls, *args, **kwargs):
             return OPTIMIZERS[cls](*args, **kwargs)
         return super().__new__(cls)
 
-    @torch.no_grad
```

Comment:
Instead, please create the decorator `no_grad` in `torch_utils.py` and import it here -- this will be more readable and will avoid redundancy

Reply:
Fixed! Imported as torch_utils to make it clear that we're not using vanilla torch.no_grad but could see the argument for just no_grad